test/parallel/test-buffer-fill.js

```diff
@@ -146,6 +146,22 @@ testBufs('yKJhYQ==', 8, 1, 'ucs2');
 testBufs('Yci0Ysi1Y8i2', 4, 1, 'ucs2');
 testBufs('Yci0Ysi1Y8i2', 12, 1, 'ucs2');
 
+// BASE64URL
+testBufs('YWJj', 'ucs2');
```

Comment:
I might be missing something, but:
1) Why is this `ucs2`?
2) How is this different from the block above, for BASE64?
3) Why is in the BASE64 block above `ucs2` is used?

Reply:
Yeah, i copied the base64 block without thinking about it much. It turns out the base64 block itself is wrong, isn't it?

Reply:
[`591242f` (#36952)](https://github.com/nodejs/node/pull/36952/commits/591242f05357235beb3c5d97959f91a08aeb12ab)